pkg/admission/initializers/initializer.go

```diff
@@ -44,6 +47,26 @@ func (i *kcpInformersInitializer) Initialize(plugin admission.Interface) {
 	}
 }
 
+// NewKubeInformersInitializer returns an admission plugin initializer that injects
+// kube shared informer factories into admission plugins.
+func NewKubeInformersInitializer(
+	kubeInformers informers.SharedInformerFactory,
+) *kubeInformersInitializer {
+	return &kubeInformersInitializer{
+		kubeInformers: kubeInformers,
+	}
+}
+
+type kubeInformersInitializer struct {
+	kubeInformers informers.SharedInformerFactory
+}
+
+func (i *kubeInformersInitializer) Initialize(plugin admission.Interface) {
+	if wants, ok := plugin.(initializer.WantsExternalKubeInformerFactory); ok {
+		wants.SetExternalKubeInformerFactory(i.kubeInformers)
+	}
+}
+
```

Comment:
this is preexisting. Why another initializer? Compae `WantsExternalKubeInformerFactory`.

Reply:
I am using upstream's `WantsExternalKubeInformerFactory` but if we want to reuse upstream's actual initializer, we have to instantiate one of https://github.com/kcp-dev/kubernetes/blob/feature-logical-clusters-1.24/staging/src/k8s.io/apiserver/pkg/admission/initializer/initializer.go, which didn't seem like the best approach.

Reply:
Nevermind, I didn't realize this was set up by default in the kube code 😄 Removing.